support-frontend/assets/components/button/button.scss

```diff
@@ -17,6 +17,8 @@
   transition: $gu-transition;
   justify-content: space-between;
   position: relative;
+  width: fit-content;
```

Comment:
worth checking there's a graceful fallback in unsupported browser since it doesn't look like support is that great: https://caniuse.com/#search=fit-content

Reply:
Thanks @joelochlann. From what I found, unless an absolute `width` or `max-width` is specified, it seems a _button used inside a form_ goes full width. As this is a reused component, I don't want to pass in any absolute widths but let's loop in @walaura to see if she has more expertise

Reply:
or maybe @SiAdcock because I think Laura is on hols

Reply:
Ah of course, it's a storybook component. Well, I can see that the reason it's going full width is because the [form is a flex container](https://github.com/guardian/support-frontend/blob/master/support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss#L452-L456). The default value of `align-items` is `stretch`, so this is why the children are full-width (try adding `align-items: flex-start` to the form style and see the difference!).

That `form--contribution` style is from our page-specific CSS though, so we definitely shouldn't change the generic storybook styling to affect it. If we really want these buttons to be left-aligned, we could put another class on those buttons and make them `align-self: flex-start`. **But** I'm not actually convinced we want them left-aligned & content-width.

Check with @ionamckendrick but I think this:

<img width="503" alt="Screenshot 2019-04-26 at 10 58 28" src="https://user-images.githubusercontent.com/5122968/56800861-fe340480-6813-11e9-989c-e5435581375e.png">

is preferable to this:

<img width="503" alt="Screenshot 2019-04-26 at 10 59 41" src="https://user-images.githubusercontent.com/5122968/56800874-05f3a900-6814-11e9-898f-47a409f274cb.png">

so no extra CSS needed to make the buttons content-width.

Reply:
I think having a fitted width is part of using the storybook components- we should try to eliminate the stylistic variance of having full-width some places and auto-width others in my understanding

Reply:
Really good one to discuss. I visually prefer full width when buttons are stacked together (the flex width looks weird) BUT I appreciate that for single CTAs the flex width is the pattern we're generally using. I'd vote for fixed width in this instance, but we should definitely pick up with Laura and a group of us to decide for the future.

Reply:
(@ionamckendrick careful with the term flex width since flex is a very loaded technical term in CSS-land! so it might not mean what you think. content-width vs parent-width is probably the least ambiguous for this discussion)

Reply:
@ionamckendrick thanks for the input and resolving this for now - will just leave them full-width for the time being